File: src/DolarArray106/D_19_MaximumAndMinimum_9/MaxMin.java
package DolarArray106.D_19_MaximumAndMinimum_9;

public class MaxMin {
    int i, max, min;

    /**
     * maximum value..
     */
    public void showMax(int[] arrayNum, int testNum) {
        max = arrayNum[0];
        for (i = 0; i < testNum; i++) {
            if (arrayNum[i] > max) { // when not bigger max then go to print.
                max = arrayNum[i]; // when get the max then print max.
            }
        }
        System.out.println("Maximum : " + max);
    }

    /**
     * minimum value..
     */
    public void showMin(int[] arrayNum, int testNum) {
        min = arrayNum[0];
        for (i = 0; i < testNum; i++) {
            if (arrayNum[i] < min) {
                min = arrayNum[i];
            }
        }
        System.out.println("Minimum : " + min);
    }
}

File: src/ProblemSolvingDayByDay/Array/KotlinArray/MaxMin.kt
package ProblemSolvingDayByDay.Array.KotlinArray

import java.util.*

object MaxMin {
    @JvmStatic
    fun main(args: Array<String>) {
        /**
         * Maximum and minimum element of array...:)
         */
        var i: Int
        val testNum: Int
        var max = 0
        var min = 0
        val array = IntArray(10)
        val sc = Scanner(System.`in`)
        print("Input the elements stored in array : ")
        testNum = sc.nextInt()

        /**
         * input array
         */
        for (i in 0..testNum) {
            print("index: $i element : ")
            array[i] = sc.nextInt();
        }

        /**
         * Max and min number..
         */
        max = array[0];
        min = array[0];
        for (i in 0..testNum) {
            if (array[i] > max) {
                max = array[i]
            } else if (array[i] < min) {
                min = array[i]
            }
        }
        println("Max : $max")
        println("Min : $min")


    }
}